tests/user/test_user_result_parser.py

```diff
@@ -229,8 +238,53 @@ def test_user_admin_passphrase_and_password_redacted_add_user_success_xml(
             result,
             TestUserConstants.TEST_ADD_USER_PASSPHRASE_AND_PASSWORD_RESULT_SUCCESS_DICTIONARY,
         )
-        self.assertNotIn(self.test_passphrase, str(result))
-        self.assertNotIn(self.test_password, str(result))
+        result_str = str(result)
+        self.assertNotIn(self.test_passphrase, result_str)
+        self.assertNotIn(self.test_password, result_str)
+        self.assertNotIn("(" + " " * (len(self.test_passphrase) + 2) + ")", result_str)
+        self.assertNotIn("(" + " " * len(self.test_password) + ")", result_str)
+
+    def test_user_admin_password_message_not_redacted_add_user_success_xml(
+        self,
+        call_racf_mock: Mock,
+    ):
+        call_racf_mock.return_value = (
+            TestUserConstants.TEST_ADD_USER_PASSWORD_RESULT_SUCCESS_XML
+        )
+        result = self.user_admin.add(
+            "squidwrd",
+            traits=TestUserConstants.TEST_ADD_USER_REQUEST_TRAITS_PASSWORD_SIMPLE,
+        )
+        self.assertEqual(
+            result,
+            TestUserConstants.TEST_ADD_USER_PASSWORD_RESULT_SUCCESS_DICTIONARY,
+        )
+        result_str = str(result)
+        self.assertNotIn("(" + self.simple_password + ")", result_str)
+        self.assertNotIn("(" + " " * len(self.simple_password) + ")", result_str)
+        self.assertIn(self.simple_password, result_str)
```

Comment:
Again, I think that checking that the result dictionary you get is the same as the sample is enough here.

Reply:
This is a fair point. There are also a lot of these for regular secrets redaction and I removed these as well. They are extraneous.